ConcernsCaseWork/ConcernsCaseWork.Tests/Pages/Case/Management/IndexPageModelTests.cs

```diff
@@ -181,6 +181,7 @@ public async Task WhenOnGetAsync_ReturnsPageModel()
 		}
 
 		[Test]
+		[Ignore("Implementation related to this test has been reversed (ticket: 89799)")]
```

Comment:
Should we delete this test all together?

Reply:
Yes I dont see why not.